examples/step-12/step-12.cc

```diff
@@ -120,7 +120,7 @@ namespace Step12
   {
     Assert(dim >= 2, ExcNotImplemented());
 
-    Point<dim> wind_field;
+    Tensor<1, dim> wind_field;
     wind_field(0) = -p(1);
     wind_field(1) = p(0);
```

Comment:
```suggestion
    wind_field[0] = -p[1];
    wind_field[1] = p[0];
```

Reply:
Just pushed the change... when you one time don't recompile the code because you think there can nothing go wrong... this happens... For the point I left the () access operator, I can change it afterwards - if this is preferred, but I would let the test run now and look if it compiles.

Reply:
We are inconsistent about using `operator()` or `operator[]` with `Point` - it isn't a big deal if we go with one or another.

Reply:
Just applied your proposal in step-12 and 12b, so using [] consistently